src/package.json

```diff
@@ -20,13 +20,13 @@
     "angular-material-data-table": "=0.10.9",
     "angular-messages": "=1.5.8",
     "babel-polyfill": "=6.9.1",
+    "bignumber.js": "^4.0.0",
```

Comment:
Sure.
I was wondering if there is a npm setup to do it for me..
And it turns out there is:
```
npm config set save-prefix '='
```